rfcs/accordians-component.md

```diff
@@ -0,0 +1,113 @@
+# Accordians Component
+
+## Usage
```

Comment:
can you add a basic example section too?

Reply:
ok